corehq/apps/reminders/models.py

```diff
@@ -1973,3 +1973,4 @@ def update_count(self, increase_by=1):
         EmailUsage.objects.filter(pk=self.pk).update(count=models.F('count') + increase_by)
 
 
+from corehq.apps.reminders import signals
```

Comment:
we need this because the signals module doesn't get imported (so the signals don't get created) unless we reference it somewhere